src/Microsoft.Health.Fhir.CosmosDb/Registration/FhirServerBuilderCosmosDbRegistrationExtensions.cs

```diff
@@ -194,6 +198,10 @@ private static IFhirServerBuilder AddCosmosDbPersistence(this IFhirServerBuilder
                 .Singleton()
                 .AsService<ICosmosClientInitializer>();
 
+            services.Add<DataPlaneCollectionSetup>()
```

Comment:
Can you move these changes up with the DataPlaneStoreProcedureInstalled, IStorageProcedureInstaller and ICollectionDataUpdater? Let's keep all these new components all together.